.gitattributes

```diff
@@ -0,0 +1,7 @@
+# See https://git-scm.com/docs/gitattributes for more about git attribute files.
+
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/210485186-76db97aa-ebe8-4db2-be21-4856e49b582c.png)

**[Optional]**

- [ ] Your commit messeges are meaningful but all your commit messages should be capitalize to make your pull request professional :+1:

Reply:
@HAMMAS-SALEEM Ok ok. I'll keep it in mind !